scriptmodules/supplementary/limelight.sh

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env bash
+
+# This file is part of RetroPie.
+# 
+# (c) Copyright 2012-2015  Florian Müller (contact@petrockblock.com)
+# 
+# See the LICENSE.md file at the top-level directory of this distribution and 
+# at https://raw.githubusercontent.com/petrockblog/RetroPie-Setup/master/LICENSE.md.
+#
+
+rp_module_id="limelight"
+rp_module_desc="Limelight Game Streaming"
+rp_module_menus="4+"
+
+function depends_limelight() {
+    getDepends oracle-java8-jdk input-utils
+}
+
+function sources_limelight() {
+    gitPullOrClone "$md_build" "https://github.com/stsfin/RetropieLimelightInstaller.git"
+    
+    wget https://github.com/irtimmer/limelight-embedded/releases/download/v1.2.2/libopus.so
+    wget https://github.com/irtimmer/limelight-embedded/releases/download/v1.2.2/limelight.jar	
+    
+    # Download limelight simple theme
+    wget https://github.com/stsfin/RetropieLimelightInstaller/releases/download/1.3.1/theme.xml
+    wget https://github.com/stsfin/RetropieLimelightInstaller/releases/download/1.3.1/limelight.png
+    wget https://github.com/stsfin/RetropieLimelightInstaller/releases/download/1.3.1/limelight_art.png
+    wget https://github.com/stsfin/RetropieLimelightInstaller/releases/download/1.3.1/limelight_art_blur.png
+}
+
+function install_limelight() {
+    md_ret_files=(
+    	'libopus.so'
+    	'limelight.jar'
+        'README.md'
+        'limelightRetroInstall.sh'
+        'limelightconfig.sh'
+        'theme.xml'
+        'limelight.png'
+        'limelight_art.png'
+        'limelight_art_blur.png'
+    )
+}
+
+function configure_limelight() { 
+    #Create romdir limelight
+    mkRomDir "limelight"
+
+    #Create limelight config script
+    cat > "$romdir/limelight/limelightconfig.sh" << _EOF_
+#!/bin/bash
+$scriptdir/retropie_packages.sh limelight configure
+_EOF_
+
+    #Install limelight simple theme
+    mkdir /etc/emulationstation/themes/simple/limelight
+    mkdir /etc/emulationstation/themes/simple/limelight/art
+    cp $md_inst/theme.xml /etc/emulationstation/themes/simple/limelight/
+    cp $md_inst/limelight.png /etc/emulationstation/themes/simple/limelight/art/
+    cp $md_inst/limelight_art.png /etc/emulationstation/themes/simple/limelight/art/
+    cp $md_inst/limelight_art_blur.png /etc/emulationstation/themes/simple/limelight/art/
+
+    #Run limelight configuration
+pushd $md_inst
+    clear
+    echo -e "\nDiscovering GeForce PC:s, when found you can press ctrl+c to stop the search, or it will take a long time \n"
+    #discover IP-addresses of Geforce pc:s
+    java -jar limelight.jar discover
+    echo -e "\n"
```

Comment:
wouldn't just echo suffice ?

first echo can probably lost the first \n and just be a normal echo.